packages/expo-cli/src/commands/generate-module/configureModule.ts

```diff
@@ -299,6 +304,64 @@ async function configureNPM(
   );
 }
 
+/**
+ * Returns a map of directiories mapped to its files, recursively from directory passed as @param dirPath, only when path contains @param containing
+ * @param dirPath Path to scan recursively
+ * @param containing Path part to look for
+ * @param result directories mapped to array containing its files
+ */
+export async function allFilesWithPathContaining(dirPath: string, containing: string) {
+  const all = await readAllFilesIn(dirPath);
+  const keys = Object.keys(all).filter(key => key.includes(containing));
+  const result: { [key: string]: string[] } = {};
+  for (let key of keys) {
+    result[key] = all[key];
+  }
+  return result;
```

Comment:
```suggestion
  return Object.entries(await readAllFilesIn(dirPath))
  	.filter(([key]) => key.includes(containing))
  	.reduce<Record<string, string[]>>((acc, [key, value]) => {
  	  acc[key] = value;
  	  return acc;
  	}, {});
```
What would you say? 😄 💅